source/common/runtime/runtime_features.cc

```diff
@@ -32,6 +32,7 @@ constexpr const char* runtime_features[] = {
     "envoy.reloadable_features.strict_method_validation",
     "envoy.reloadable_features.new_http1_connection_pool_behavior",
     "envoy.reloadable_features.new_http2_connection_pool_behavior",
+    "envoy.reloadable_features.ext_authz_http_service_disable_lowercase_string_matcher",
```

Comment:
Sorry I'm confused. This is true by default, and isn't the default behavior to not have a lower case string matcher? So would a better name be enable_lowercase_string_matcher?

Reply:
Since it is enabled by default: to "disable lowercase string matcher" is the feature to be enabled. Yeah, it is confusing, should we change as "enable_always_case_sensitive_string_matcher"?

Reply:
The new default behavior is to *not* use the old lower case string matcher, right? Then yeah, I would probably do something like: `"envoy.reloadable_features.ext_authz_http_service_enable_case_sensitive_string_matcher` ?